packages/telemetry/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import { TelemetryAdaptersConfig } from "@scramjet/types";
+import { LokiAdapter } from "./adapters/loki";
```

Comment:
Food for thoughts here: we can't do tree-shaking when we don't use the specific adapter - this is equally true for for instance kubernetes adapter - it's not loaded, so memory is not taken there, but we still need to ship it with the libs... Takes up disk space at least, the above takes also memory.

Reply:
we can split these modules, `adapters` as well, into smaller ones and install them at runtime (at startup) if they are needed